config.tutorial.yaml

```diff
@@ -91,6 +91,11 @@ enable:
   build_cutout: false
   build_natura_raster: true  # If True, then build_natura_raster can be run
 
+# definition of the Coordinate Reference Systems
+crs:
+  default_crs: EPSG:4326  # general geographic projection, not used for metric measures
+  metric_crs: EPSG:3085  # projection for metric measures only
```

Comment:
REALLY COOL and well needed! One note:

There are projection that preserve angles, preserve areas and distances. Usually, there is not a one solution fits all case: https://learn.arcgis.com/en/projects/choose-the-right-projection/

Good for **preserving and calculating angles**:
- https://epsg.io/4326

Suitable for **preserving and calculating distances**:
- https://epsg.io/3857 aka. wgs 84 is the OpenStreetMap and Google Maps standard

Ok, for **preserving and caluclating areas**:
- we can go for https://epsg.io/3085 , but it's scope is for AMERICA. Maybe good

Reply:
So maybe we should define this three options in the config.yaml?

Reply:
Yeah, the 3085 was supposed to be 3857! A refuse, thanks!
Anyway, I wanted to add also some suggestions of alternative values

Reply:
> So maybe we should define this three options in the config.yaml?

The 3085 should not be used unless the user focuses on America only.
The default values shall have global scope